pkg/chartmuseum/server/multitenant/api.go

```diff
@@ -19,23 +19,40 @@ package multitenant
 import (
 	"fmt"
 	pathutil "path/filepath"
+	"sort"
 
 	cm_logger "helm.sh/chartmuseum/pkg/chartmuseum/logger"
 	cm_repo "helm.sh/chartmuseum/pkg/repo"
 
 	helm_repo "k8s.io/helm/pkg/repo"
 )
 
-func (server *MultiTenantServer) getAllCharts(log cm_logger.LoggingFn, repo string) (map[string]helm_repo.ChartVersions, *HTTPError) {
+func (server *MultiTenantServer) getAllCharts(log cm_logger.LoggingFn, repo string, offset int, limit int) (map[string]helm_repo.ChartVersions, *HTTPError) {
 	indexFile, err := server.getIndexFile(log, repo)
 	if err != nil {
 		return nil, &HTTPError{500, err.Message}
 	}
-	return indexFile.Entries, nil
+	if offset == 0 && limit == -1 {
+		return indexFile.Entries, nil
+	}
```

Comment:
It is only possible to have the default existing behavior by programatically specifying `0, -1` for the offset and limit parameters. This is not possible from the current REST routes.

Reply:
@davidovich Please check the changes now. Fixed this by setting default limit to "-1" in get all charts api